### DISCUSSION TITLE
Ensuring Fair Pay

### DISCUSSION THREAD

**Discussion by subsoap** [WICG/webmonetization#210] [General]
I can see a potential problem in the future where even if developers use receipt validation they can still be cheated if users use intentionally dodgy providers who pay out the absolute minimum possible for WM to be active. So I believe there should be some standard in the amount in value a single payment can be allowed to go down to, and a reasonable expectation of minimum value over a period of time. If these standards are not met it should be fair for services which monetize through WM to say that the WM payments are too low over too long of a period of time and thus are invalid to count for unlocks/benefits.

So if I have a game that has WM enabled and a player is playing for 5 hours in a day but they only sent 1cent USD value total then it should be fair for us to mark them as a non-payer on probation until a minimum amount of value is paid and it stays above that minimum over a long enough period of time (or ban them from WM payments and only accept other forms of payment). Of course this can be tricky to do with the way content pays out and it may frustrate users when their monthly value is tapped out (so that should also be possible to more clearly communicate by providers to their subscribers, and offer ways to top up their value if they are nearly the point of not having an active WM minimum) but it's important to still be fair to those who implement WM.

Minimums of value should be communicated to potential providers well, and potentially providers which do not pay fair should not be listed officially. 

This needs more discussion, it will be a legitimate concern for people in the future.

**Comment by coderofsalvation** [comment]
I think this is a fair point.
A race to the bottom is inevitable, unless prevented inside the protocol.
But..once prevented, it will make 'the free web' more attractive again.
I guess the flattr project also has lots of knowledge on this chicken-egg problem (btw. the egg was first).

**Comment by adrianhopebailie** [comment]
This is a difficult problem to solve in an open ecosystem.

Making the rate "negotiable" will lead to a race to the bottom and providers paying the bare minimum. That's our rationale so far for paying a flat rate.

Websites should not rely on all providers paying the same rate and should decide what is a fair minimum for their service. If users keep getting cut-off from a service because their provider is under-paying they'll either find a cheaper service or complain to their provider.

I think there is a lot to still learn here as this evolves.

**Comment by coderofsalvation** [comment]
I truly respect the fact that this project is trying to tackle it, and I hope it will.
Good luck!

**Reply by BGAK2024** [reply]
Any ideas already in works
